main.js

```diff
@@ -0,0 +1,57 @@
+var boardArr = [];
+
+var board = {
+
+  initializeGame: function() {
+    boardArr = [ ['5','4','3','2','1'], [], [] ];
+    console.log('Game begin, board is:');
+    this.printBoard();
+  },
+
+  printBoard: function() {
+    var pegDisplay =  boardArr.map(function(peg) {
+      return ('--- '+peg.join(' '));
+     })
+     console.log(pegDisplay[0]);
```

Comment:
Instead of logging each peg, you could use the `reduce` method to console log one string.